modules/tooling-support-parent/test-extension/src/test/java/org/mule/runtime/module/tooling/TestExtensionDeclarationUtils.java

```diff
@@ -310,15 +323,36 @@ public static OperationElementDeclaration multiLevelShowInDslGroupOPDeclaration(
     if (country != null) {
       parameterGroupElementDeclarer.withParameter("country", ParameterSimpleValue.of(country));
     }
+    if (city != null) {
+      parameterGroupElementDeclarer.withParameter("city", ParameterSimpleValue.of(city));
+    }
 
-    if (continent != null || country != null) {
+    if (continent != null || country != null || city != null) {
       elementDeclarer.withParameterGroup(parameterGroupElementDeclarer.getDeclaration());
     }
     return elementDeclarer.getDeclaration();
+
   }
 
   public static ComponentElementDeclaration<?> invalidComponentDeclaration() {
     return TEST_EXTENSION_DECLARER.newConstruct("invalid").getDeclaration();
   }
 
+  public static ComponentElementDeclaration<?> componentDeclarationWrongConfigRef() {
```

Comment:
I wouldn't add this type of methods here. In any case, add a method that accepts the configName you want to use. 
But this one seems too specific for the test case you are trying. 
At least the idea of this class, was to provide an easy way to declare all elements of the TestExtension. But not to test different cases. That should be the concern of the user class

Reply:
Ok, I will change it.